src/actions/index.js

```diff
@@ -0,0 +1,59 @@
+export const select = (user) => {
+  return {
+    type: "USER_SELECTED",
+    payload: user
+  }
+};
+
+export const unSelect = () => {
+  return {
+    type: "USER_NOT_SELECTED",
+    payload: null
+  }
+};
+
+export const addUser = (name, surname, sex) => {
+  return {
+    type: "ADD_USER",
+    name: name,
```

Comment:
Good practice to have payload always when you send data

`payload: {
  name,
  sex,
  surname
}`

This way it's easier to get data inside reducer, 

Also use destruction 

Review this across your actions